recipe/meta.yaml

```diff
@@ -22,7 +22,7 @@ requirements:
     - pip
     - poetry-core >=1.0.0
   run:
-    - python >=3.6,<3.10
+    - python >=3.6,<3.11
```

Comment:
You need to bump the build number too. Also, each PR produces a new package, that means we have two build number 0 that have the wrong python upper limit!

Reply:
Hi Filipe - thanks - I did figure out that bump that other python dependency.  I didn't see it the first time.  I bumped the build, pushed to pypi and hoping to see the pull request here soon.  Thanks for commenting.

Reply:
Note that, b/c the build number was not bumped, only the one from the first PR was uploaded, see https://anaconda.org/conda-forge/gdptools/files?version=0.0.26

That means the conda package is still only installable in python>=3.6,<3.10. We need a new PR that bumps the build number from 0 to 1.

Reply:
@ocefpaf Ok - thanks Filipe.  So you are saying that if I make a mistake as in this case.  I can correct the meta.yaml for example, and push the build number from 0 to 1, and it will rebuild the same version but with the corrections?

Reply:
Yep. And, if the error is a breaking one, not the case here b/c the package does work in the narrower Python limit, we also need to mark them as broken. See https://conda-forge.org/docs/maintainer/updating_pkgs.html#removing-broken-packages

Reply:
Thanks again @ocefpaf - I very much appreciate the help.